config/jobs/etcd/etcd-presubmits.yaml

```diff
@@ -118,7 +118,6 @@ presubmits:
 
   - name: pull-etcd-integration-1-cpu-amd64
```

Comment:
Do we want to include the other integration jobs in this pr as well?

Reply:
I'd like to add the others as optional. I considered doing it in a follow-up PR to keep the PR more atomic. However, if you think it's okay, I can add them to this PR.

Reply:
Let's bring them in as optional here then just one more pr to tune them later 👍🏻

Reply:
Done.